x-pack/plugins/security_solution/public/resolver/store/actions.ts

```diff
@@ -24,6 +24,35 @@ interface UserBroughtProcessIntoView {
   };
 }
 
+/**
+ * Dispatched to notify state that a different panel needs to be displayed
+ */
+interface AppDisplayedDifferentPanel {
```

Comment:
If we merge these actions let's schedule an immediate follow up PR to refactor this.

* AppDisplayedDifferentPanel
* AppDetectedNewIdFromQueryParams
* AppDetectedMissingEventData